python/foglamp/services/core/api/update.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" FogLAMP package updater API support"""
+
+from aiohttp import web
+import uuid
+import asyncio
+import json
+
+# FogLAMP imports
+from foglamp.services.core import server
+from foglamp.common import logger
+
+_LOG_LEVEL_DEBUG = 10
+_logger = logger.setup(__name__, level=_LOG_LEVEL_DEBUG)
+
+__author__ = "Massimiliano Pinto"
+__copyright__ = "Copyright (c) 2018 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+_help = """
+    -----------------------------------------
+    | PUT             | /foglamp/update     |
+    -----------------------------------------
+    | no paylod       |                     |
+    -----------------------------------------
+"""
+
+###
+# The update.py is part of the FogLAMP REST API:
+# when called via PUT /foglamp/update it will add the "update task"
+# to scheduler queue for execution.
+#
+# This only happens if the 'FogLAMPUpdater' is found
```

Comment:
Not sure about the specs, FoglampSnapUpdater will look more intuitive.

Reply:
FogLAMPUpdater is a generic name.